flake.nix

```diff
@@ -84,7 +92,7 @@
             # tools we want in our shell, from hackage
             tools =
               {
-                cabal = "3.12.1.0";
+                cabal = "3.10.3.0";
```

Comment:
Same here, how come we need to downgrade cabal?

Reply:
This seems to work.